auth_matrix/templates/auth_matrix/admin/includes/object_tools_items.html

```diff
@@ -0,0 +1,8 @@
+{% load i18n %}
+{% if can_view_auth_matrix %}
+    <li>
+        <a href="{% url 'auth_matrix:authorization_matrix' %}" class="button mt">
```

Comment:
one advantage of adding the URLs via the admin mixin, is that we will actually be able to use a URL name that doesn't have namespace issues :)

Reply:
I'm not sure that I have understood what you meant here.
Isn't this already making use of the changes we made ?

Reply:
django has some mechanisms making it possible to load an app multiple times with different namespaces. however, reversing URLs requires you to know all the namespaces upfront, which is of course not possible in a library. users may include the url patterns in different ways, but that breaks the app because then it cannot reverse the URL name anymore

All of these problems should go away automagically when this url/view is part of the admin views through the `admin.ModelAdmin.get_urls` method, since it takes care of properly resolving the app and we don't have to worry about it. So then you'll use `admin:user_authorization_matrix` for example (if that's the URL name that we'll go with).

Reply:
I don't get this. The authorization matrix works for both groups and user. Attaching the url to both model wouldnt allow me to use the same url pattern on both models. Having different urls would't allow me to include the `object_tools_items.html ` template. I don't think I understand the suggestion.

Reply:
Eventually, this seems wrong to me but this work :

```
class DummyModel(models.Model):
    pass

    class Meta:
        managed = False
        verbose_name = 'My Model'
        verbose_name = 'My Model'


class DummyModelAdmin(admin.ModelAdmin):
    pass

    def get_urls(self):
        from django.urls import path

        return [
            path(
                "matrix/",
                self.admin_site.admin_view(AuthorizationMatrixView.as_view()),
                name="authorization_matrix",
            )
        ]


admin.site.register(DummyModel, DummyModelAdmin)
```

Reply:
ok - discussed together